float64[] joint_pos
---
bool state
